<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteCreator</class>
 <widget class="QMainWindow" name="NoteCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="wordCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Word Count</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="writeSpace">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>551</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 5px, bold black</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel
 {
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
       </property>
       <property name="text">
        <string>Tools</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="fontComboBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>201</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
       </property>
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="commntTextEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>11</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="noteNameLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
     </property>
     <property name="text">
      <string>Note Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="noteNameLine"/>
    <widget class="QLabel" name="noteMajorLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
    border: 2px solid darkkhaki;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
     </property>
     <property name="text">
      <string>Major</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="noteMajorLine"/>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>wordCountLabel</zorder>
   <zorder>writeSpace</zorder>
   <zorder>groupBox</zorder>
   <zorder>noteNameLabel</zorder>
   <zorder>noteMajorLabel</zorder>
   <zorder>noteNameLine</zorder>
   <zorder>noteMajorLine</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_As_Online"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPaste"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menuABout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuText">
     <property name="title">
      <string>Text</string>
     </property>
     <addaction name="actionUnderline"/>
     <addaction name="actionItalics"/>
     <addaction name="actionBold"/>
     <addaction name="separator"/>
     <addaction name="actionFont"/>
     <addaction name="actionFont_Color"/>
     <addaction name="actionHighlight_Color"/>
    </widget>
    <addaction name="menuText"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuABout"/>
   <addaction name="menuFormat"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen_File"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionCut"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUnderline"/>
   <addaction name="actionItalics"/>
   <addaction name="actionBold"/>
   <addaction name="actionFont_Color"/>
   <addaction name="actionHighlight_Color"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/Paste-icon.png</normaloff>:/Icons/Paste-icon.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/Copy-icon.png</normaloff>:/Icons/Copy-icon.png</iconset>
   </property>
   <property name="text">
    <string>Copy All</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/save-icon.png</normaloff>:/Icons/save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/File-New-icon.png</normaloff>:/Icons/File-New-icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/Apps-Actions-Cut-icon.png</normaloff>:/Icons/Apps-Actions-Cut-icon.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/File-Open-icon.png</normaloff>:/Icons/File-Open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/Save-as-icon.png</normaloff>:/Icons/Save-as-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save As.. Locally</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/text-underline-icon.png</normaloff>:/Icons/text-underline-icon.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionItalics">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/text-italic-icon.png</normaloff>:/Icons/text-italic-icon.png</iconset>
   </property>
   <property name="text">
    <string>Italics</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/text-bold-icon.png</normaloff>:/Icons/text-bold-icon.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionFont_Color">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/Icons/font-colors-icon.png</normaloff>:/Icons/font-colors-icon.png</iconset>
   </property>
   <property name="text">
    <string>Font Color</string>
   </property>
  </action>
  <action name="actionHighlight_Color">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/Icons/Actions-color-picker-black-icon.png</normaloff>:/pics/Icons/Actions-color-picker-black-icon.png</iconset>
   </property>
   <property name="text">
    <string>Highlight Color</string>
   </property>
  </action>
  <action name="actionSave_As_Online">
   <property name="text">
    <string>Save As..Online</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
